packages/eslint-plugin/docs/rules/dependencies-docblocks.md

```diff
@@ -0,0 +1,36 @@
+# Enforce dependencies docblocks formatting (dependencies-docblocks)
+
+Ensures that all top-level package imports adhere to the dependencies grouping conventions as outlined in the [Coding Guidelines](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#imports).
+
+Specifically, this ensures that:
+
+- An import is preceded by "External dependencies", "WordPress dependencies", or "Internal dependencies" as appropriate by the import source.
+
+## Rule details
+
+The following patterns are considered warnings:
```

Comment:
Perhaps we could use incorrect/correct instead of `warning` here, so as to prevent confusion with the error/warning setting of the rule?

```suggestion
The following patterns are considered **incorrect** code:
```

Reply:
I made it this way thinking I was adhering to the format set forth by ESLint, but I guess at some point they must have changed from a pattern of:

>The following patterns are considered problems:

To:

>Examples of **incorrect** code for this rule:

https://github.com/eslint/eslint/commit/8c299469ed11a65fa7c51a5d94e6f269402020c3#diff-61fab3ca0b432c4678c6546a292fe58f

I can update accordingly.